ui/inspectors/volume.reel/volume.html

```diff
@@ -100,9 +100,9 @@
                 "element": {"#": "allocationBar"}
             },
             "bindings": {
-                "totalSize": {"<-": "+@owner.object.properties.size.rawvalue"},
-                "usedSize": {"<-": "+@owner.object.properties.allocated.rawvalue"},
-                "available": {"<-": "+@owner.object.properties.free.rawvalue"}
+                "totalSize": {"<-": "@owner.rootDataset.properties.available.parsed + @owner.rootDataset.properties.used.parsed"},
```

Comment:
The `rawvalue` property is a string, so we needed to convert it to a number. Since then the middleware introduced the `parsed` property which for size properties is a number.